src/ledger/accountinfo.ts

```diff
@@ -1,31 +1,12 @@
 import {validate, removeUndefined, dropsToXrp} from '../common'
+import {RippleAPI} from '../api'
+import {AccountInfoResponse} from '../common/types/commands/account_info'
 
-type AccountData = {
-  Sequence: number,
-  Account: string,
-  Balance: string,
-  Flags: number,
-  LedgerEntryType: string,
-  OwnerCount: number,
-  PreviousTxnID: string,
-  AccountTxnID?: string,
-  PreviousTxnLgrSeq: number,
-  index: string
-}
-
-type AccountDataResponse = {
-  account_data: AccountData,
-  ledger_current_index?: number,
-  ledger_hash?: string,
-  ledger_index: number,
-  validated: boolean
-}
-
-type AccountInfoOptions = {
+type GetAccountInfoOptions = {
   ledgerVersion?: number
 }
 
-type AccountInfoResponse = {
+type GetAccountInfoFormattedResponse = {
```

Comment:
I see that some types were renamed, like `AccountInfoResponse` -> `GetAccountInfoFormattedResponse`, `TrustlinesOptions` -> `GetTrustlinesOptions`, `Trustline` -> `ParsedTrustline`. Could you explain a bit about how these types should be named?

Reply:
Definitely, a would love some feedback from you on these as well.

The main idea behind the type renaming/reorganization is to best support the rippled API as the main source of truth. But the current `AccountInfoResponse`, `TrustlinesOptions`, and `Trustline` types all describe special objects created and defined specifically by ripple-lib. So, for example, now that we're adding typing info for the actual standard "account_info" response format, `AccountInfoResponse` is a confusing name for the custom/specialized `getAccountInfo()` response  object. Same with the current `Trustline`, which is a special custom parsing of a trustline and not the standard API format for one.

What do you think? So far I haven't had followed any specific rules while renaming existing custom types that conflicted with standard formats. But happy to work out a set of rules to follow if that's a concern.